### ISSUE TITLE
Lion King (Genesis), Keyboard, L/R input

### ISSUE THREAD

**Issue by haggerlfd** [OpenEmu/OpenEmu#4319]
Using an early 2015 Macbook Air with Catalina 10.15.7
Using the latest official version of OpenEmu (v2.2.1)
The issue I will outline was present in older versions of the emulator as well

Emulating The Lion King (Genesis) and playing using a KEYBOARD. The issue I will outline persists after testing with different keyboards.

The issue, however, does not occur when using a ps4 controller to play the game

While playing The Lion King (Genesis), pressing the left and right directional buttons simultaneously on the keyboard causes the character to shift rapidly between those directions if the character is in midair (L/R input or L/R bug/glitch basically).

This is a banned feature in speedrunning so I can't speedrun this game using OpenEmu. One is likely to accidentally press both the left and right buttons simultaneously while shifting directions, and thus cause this glitch to occur during a speedrun.

This glitch is not possible on an original Sega Genesis/Megadrive using a standard non-turbo controller.

Interestingly, if both the left and right buttons are pressed simultaneously while the character is on the ground, the character responds to the button that is pressed first. Moreover, the character changes direction if it runs into a wall while both directional buttons are pressed. 

The same issue persists if I use the standalone Genesis Plus emulator to emulate The Lion King (Genesis) as well. 

Some points that I think are worth considering: 

1. This issue does not occur on Disney's Aladdin (Genesis), another game I emulate. There the character just goes to the right if I press both directional buttons simultaneously, whether the character is on the ground or in midair.

2. The issue does not occur on Mickey's Ultimate Challenge (Genesis). There, if I press both directional buttons simultaneously, the character always goes to the right if it is in midair, and it moves in the direction of the button that is pressed first if it is on the ground.

3. The issue does not happen with the SNES version of The Lion King. On The Lion King (SNES), the character always goes to the left if I press both directional buttons simultaneously, whether the character is on the ground or in midair. This is with the SNES9x core.

4. If you play the SNES version of The Lion King with the BSNES core, then instead of going to the left, the character does not move in any direction when both directional buttons are pressed together. The character just stays still, whether on the ground or in midair.

I tested all this with multiple keyboards.

Basically the issue is so specific that it baffles me and I can't find a solution for it anywhere. It only happens with this one game (The Lion King Genesis), and only when playing it with a keyboard.

I am accustomed to playing these games with a keyboard after doing it for several years. I cannot play them well, let alone speedrun them, if I use a ps4 controller.

The emulator should cause the character to move in either one of those directions (left or right) when the directional keys are pressed together, just like it does with other games.
